src/main/java/org/sonarlint/intellij/actions/SonarLintToolWindow.java

```diff
@@ -469,5 +473,20 @@ private Content getTaintVulnerabilitiesContent() {
   @Override
   public void bindingChanged(@Nullable ProjectBinding previousBinding, @Nullable ProjectBinding newBinding) {
     runOnUiThread(project, this::refreshViews);
+
+    showSharedConfigurationNotification(project, "Project successfully bound with \"" +
+        newBinding.getProjectKey() + "\" on \"" + newBinding.getConnectionName() +
+      "\"." +
+        " If you share this configuration, a file will be created in this working directory," +
+        " making it easier for other team members to configure the binding for the same project." +
+        "You may also decide to share this configuration later from your list of bound projects.", "ShareConfigurationAction",
+      new ShareConfigurationAction("Share configuration", CONNECTED_MODE_BENEFITS_LINK, null));
+  }
+
+  public void showSharedConfigurationNotification(Project project, String message, String doNotShowAgainId, AnAction action) {
```

Comment:
I don't think you need to pass the action as parameter if it's always the same one, same for the `doNotShowAgainId`